slave.mk

```diff
@@ -549,8 +549,13 @@ $(SONIC_INSTALL_DEBS) : $(DEBS_PATH)/%-install : .platform $$(addsuffix -install
 		$(foreach deb, $($*_CONFLICT_DEBS), \
 			{ while dpkg -s $(firstword $(subst _, ,$(basename $(deb)))) | grep "^Version: $(word 2, $(subst _, ,$(basename $(deb))))" &> /dev/null; do echo "waiting for $(deb) to be uninstalled" $(LOG); sleep 1; done } )
 		# put a lock here because dpkg does not allow installing packages in parallel
+		# this lock didn't avoid all parallel installing. Add some retry.
```

Comment:
why this lock did not avoid all parallel installing? what missed?

Reply:
From Guohan's PR, we can see it is the following package holding the lock:
src/openssh/Makefile:22:        sudo http_proxy=$(http_proxy) apt-get -y build-dep openssh

Maybe adding retry is still the best way to resolve this issue?